tests/test_jump.py

```diff
@@ -485,33 +484,24 @@ def test_find_faint_extended_sigclip():
     assert np.all(gdq[0, 1, 12:23, 24] == 0)
     assert np.all(gdq[0, 1, 12:23, 13] == 0)
 
-    #  Check that the flags are not applied in the 3rd group after the event
+    # Check that the flags are not applied in the 3rd group after the event
     assert np.all(gdq[0, 4, 12:22, 14:23]) == 0
 
 
-def test_inside_ellipse5():
```

Comment:
Seeing `def test_inside_ellipse5()` and `def test_inside_ellipes5()` was really confusing for me. Also lots of code duplication. So I parametrized these tests.